static/cdn/maintenance.html

```diff
@@ -1,208 +1,329 @@
+							if (showingElements && showingElements.length) {
+								for (var j = 0; j < showingElements.length; j++) {
+									showingElements[j].style.display = 'none';
+								};
+							}
+						};
+					}
+				}
+			};
+			// wait until document loaded to assign onclick handlers
+			if (document.readyState === "complete" ||
+				(document.readyState !== "loading" && !document.documentElement.doScroll)) {
+				onLoaded();
+			} else {
+				document.addEventListener("DOMContentLoaded", onLoaded);
+			}
+	  </script>
   </head>
   <body>
-    <div class="box" id="page-maint">
-      <div class="box-head">
-      </div>
-      <div class="box-content">
-        <div class="twitter-containter">
-          <img src="http://scratch-maintenance.s3.amazonaws.com/site-down.png" style="margin-left:100px;padding:10px;float:left;"/>
-          <div class="twitter-feed">
-            <a class="twitter-timeline" href="https://twitter.com/scratch" height="200" width="400" data-widget-id="534361794572058626" data-chrome="noheader nofooter transparent noborders" data-tweet-limit="1">Tweets by @scratch</a>
-            <script>!function(d,s,id){var js,fjs=d.getElementsByTagName(s)[0],p=/^http:/.test(d.location)?'http':'https';if(!d.getElementById(id)){js=d.createElement(s);js.id=id;js.src=p+"://platform.twitter.com/widgets.js";fjs.parentNode.insertBefore(js,fjs);}}(document,"script","twitter-wjs");</script>
-          </div>
-        </div>
-        <div class="box-header">
-            <h1>The Scratch Team is making some changes to the website.</h1>
-          </div>
-          <h3>
-            Try reloading this page in a few minutes.
-            <span class="icon-reload" onclick="location.reload(true);"></span>
-          </h3>
-        <p>You can still <a href="https://download.scratch.mit.edu/scratch2download/">Download Scratch</a> and make projects.</p>
-        <img src="http://scratch-maintenance.s3.amazonaws.com/scratch.png" style="margin:20px; padding:10px;">
-      </div>
-    </div>
+		<div class="page">
+			<div class="header-wrapper">
+				<div class="header-content">
+					<img src="http://scratch-maintenance.s3.amazonaws.com/logo_sm.png" width="80px" />
+				</div>
+			</div>
+			<div class="main">
+				<div class="content">
+					<h1>The Scratch Team is working hard to fix an issue with the Scratch website</h1>
+					<h2 class="reload" onclick="location.reload()">Try reloading the page in a few minutes</h2>
+					<h2 class="comment">Check <a href="https://twitter.com/scratch">@Scratch</a> for updates</h2>
+					<hr>
+					<div class="narrow">
+						<p>While the website is being fixed, you can use the Scratch app to make projects offline</p>
+					</div>
+					<button class="toggleShowing showing"><span>Download Scratch app</span></button>
+					<div class="grid-wrapper hidden">
+					  <div class="grid-content">
+					    <div class="grid-box">
+								<div class="os-name-row">
```

Comment:
Nitpick, but are these indented too far?

Reply:
Thanks for noticing -- some of my spacing was tabs, and some was spaces. I'm now converting it all to spaces.